proto/route_rule.proto

```diff
@@ -19,11 +19,12 @@ message RouteRule {
     // perform PA validation in the mapping table belonging to vnet_name. Default is set to true
     optional bool pa_validation = 5;
     // Optional
-    // Metering class-id
-    optional uint64 metering_class = 6;
+    // Metering class aggregate bits
+    optional uint32 metering_class_or = 6;
+    optional uint32 metering_class_and = 7;
     // Optional
     // optional region_id which the vni/prefix belongs to as a string for any vendor optimizations
-    optional string region = 7;
+    optional string region = 8;
```

Comment:
is this field needed? looks like we don't have this field in SAI.

Reply:
This field is present in the HLD, should we remove it from there if it's unneeded?